SourceX/controls/game_controls.cpp

```diff
@@ -98,14 +101,22 @@ bool GetGameAction(const SDL_Event &event, ControllerButtonEvent ctrl_event, Gam
 	if (!in_game_menu) {
 		switch (ctrl_event.button) {
 		case ControllerButton_BUTTON_LEFTSHOULDER:
+#ifdef __vita__
+			if ((select_modifier_active && !getIniBool("vita","switch_potions_and_clicks")) || (getIniBool("vita","switch_potions_and_clicks") && !select_modifier_active) ) {

```

Comment:
1. These probably shouldn't be vita-specific
2. `ini` settings should be read once at startup

Reply:
Should i make `dpad_hotkeys` non-vita specific too?

Reply:
Yeah I think so. It's a useful option for any controller with a left analog stick.